openghg/util/_download.py

```diff
@@ -36,7 +36,7 @@ def download_data(
     from tqdm.auto import tqdm  # type: ignore
     from urllib.parse import urlparse
     from requests.adapters import HTTPAdapter
-    from urllib3.util.retry import Retry
+    from urllib3.util.retry import Retry  # type: ignore
```

Comment:
Does this pass with mypy if `types-urllib3` is installed?

Reply:
I ran a pip install for `types-urllib3` but I do see this error when I run `mypy` (locally):
```
(openghg_env) [rt17603@bp1-login02 openghg]$ mypy openghg
openghg/util/_download.py:57: error: Argument "max_retries" to "HTTPAdapter" has incompatible type "urllib3.util.retry.Retry"; expected "Union[requests.packages.urllib3.util.retry.Retry, int, None]"
Found 1 error in 1 file (checked 96 source files)
```
Not sure if this the same error as previously though.

Reply:
Ah OK, maybe we can leave that ignore for now and I'll have a look sometime.